contrib/cf-doc/get-meta-promises.jq

```diff
@@ -0,0 +1,3 @@
+#!/bin/jq -f
+
+.bundles[]|{bundle: .name,file: .sourcePath,toBeFiltered: (.promiseTypes[]|select(.name == "meta")|.contexts[]|{context: .name,promises: .promises[]})}|{bundle,file,context: .toBeFiltered.context,promiser: .toBeFiltered.promises.promiser,metacomment: (.toBeFiltered.promises.attributes[]|select(.lval == "string")|.rval.value),linenumber: (.toBeFiltered.promises.attributes[]|select(.lval == "string")|.line)}
```

Comment:
Is it possible to add some kind of comment explaining what it's doing? Sample input/output and maybe text version, like:
> for each item in "bundles", build object with name and file and temporary toBeFiltered object holding meta information (object with name and array of promises for each item in "contexts" build).

(if I read it right, there might be some intermediate result useful for understanding this command)
> Then, take resulting bundle objects and reconstruct them to extract context and promiser from the toBeFiltered object together with metacomment which is `.rval.value` of `promises.attribute` element with `.lval == "string"` and a linenumber where it happened.

Maybe something more human readable is possible...